src/org/usfirst/frc2175/command/group/RunIntakeInGroup.java

```diff
@@ -12,13 +13,20 @@
  */
 public class RunIntakeInGroup extends CommandGroup {
 
+    private double sideBeltSpeed;
+    private double mainBeltSpeed;
```

Comment:
These variables are actually never used! Eclipse warns you about this. I'll explain below.